apcd_cms/src/apps/utils/apcd_database.py

```diff
@@ -852,6 +852,42 @@ def create_threshold_exception(form, exception, sub_data):
             conn.close()
 
 
+# def get_cdl_exceptions(file_type):
```

Comment:
Possibly remove this from code base and put here as depreciated business logic? 
[https://tacc-main.atlassian.net/wiki/spaces/UP/pages/66617347/APCD+Database+and+Business+Logic](https://tacc-main.atlassian.net/wiki/spaces/UP/pages/66617347/APCD+Database+and+Business+Logic)